app/Http/Controllers/API/UserController.php

```diff
@@ -0,0 +1,273 @@
+<?php
+
+namespace App\Http\Controllers\API;
+
+use App\Http\Controllers\Controller;
+use App\Models\Area;
+use App\Models\Endorsement;
+use App\Models\User;
+use Illuminate\Database\Eloquent\Builder;
+use Illuminate\Http\Request;
+use Illuminate\Support\Collection;
+
+class UserController extends Controller
+{
+    /**
+     * Get data based on GET request
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index(Request $request)
```

Comment:
```suggestion
    public function index(Request $request): \Illuminate\Http\Response
```
suggestion: let's clarify the comment or remove it, it says almost nothing about functionality.

Reply:
Kind of agree, but we've documented all functions in CC that way, so I'll look into improving the comment and input typ,es